libs/dh/core/shell/src/lib/dh-core-shell.module.ts

```diff
@@ -135,6 +135,7 @@ const routes: Routes = [
       useClass: MsalInterceptor,
       multi: true,
     },
+    dhAuthorizationInterceptorProvider,
```

Comment:
suggestion: let's add a comment about the place in which this interceptor is provided since it's important to run interceptors in specific order.

Reply:
Fixed